tests/measurements/test_classical_shadow.py

```diff
@@ -821,8 +874,42 @@ def test_return_distribution(wires, interface, circuit_basis, basis_recipe):
     assert np.allclose(new_ratios2, 1 / 2, atol=1e-1)
 
 
+@pytest.mark.parametrize("wires", [1, 3])
+@pytest.mark.all_interfaces
+@pytest.mark.parametrize("interface", ["numpy", "autograd", "jax", "tf", "torch"])
+@pytest.mark.parametrize("circuit_basis, basis_recipe", [("x", 0), ("y", 1), ("z", 2)])
+def test_availability_legacy_arithmetic(wires, interface, circuit_basis, basis_recipe):
+    """Test that the legacy arithmetic can run. No check with results"""
+    # high number of shots to prevent true negatives
+    shots = 100
+
+    dev = DefaultQubitLegacy(wires=wires, shots=shots)
+
+    @qml.qnode(dev, interface=interface)
+    def circuit():
+        for wire in range(wires):
+            if circuit_basis in ("x", "y"):
+                qml.Hadamard(wire)
+            if circuit_basis == "y":
+                qml.RZ(np.pi / 2, wire)
+
+        return qml.classical_shadow(wires=range(wires))
+
+    tape = qml.workflow.construct_tape(circuit)()
+    obs = tape.observables[0]
+    legacy_bits, legacy_recipes = dev.classical_shadow(obs=obs, circuit=circuit)
```

Comment:
Already have a test here, why the legacy device classical shadow still not covered in the report??

Reply:
Solved: DQL redefined its own classcial_shadow. The `QubitDevice.classical_shadow` never called.